bootstrap/simple-errors.php

```diff
@@ -0,0 +1,13 @@
+<?php
+
+function displaySimpleError($tite, $code, $message)
+{
+	$error_display = file_get_contents(__DIR__ . '/../simple_error_template.html');
+	$replacing = array(
+		'$tite' => $title,
```

Comment:
Typo?

Reply:
Damn Sonarcloud is pretty good too. :sweat_smile: